docs/Api/CatalogApi.md

```diff
@@ -48,7 +48,7 @@ try {
 
 Name | Type | Description  | Notes
 ------------- | ------------- | ------------- | -------------
- **body** | [**\SquareConnect\Model\BatchDeleteCatalogObjectsRequest**](../Model/BatchDeleteCatalogObjectsRequest.md)| An object containing the fields to POST for the request.  See the corresponding object definition for field details. |
+ **body** | [**\SquareConnect\Model\BatchDeleteCatalogObjectsRequest**](../Model/\SquareConnect\Model\BatchDeleteCatalogObjectsRequest.md)| An object containing the fields to POST for the request.  See the corresponding object definition for field details. |
```

Comment:
Addressed in https://github.com/square/connect-php-sdk/pull/25/commits/b1fe1df9723b37ba9a547676371a6c049f661692